src/style.css

```diff
@@ -1,3 +1,151 @@
+html {
+  .roboto-thin {
+    font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts. In case the main font doesn't load, you should have a generic font-family as a backup. For example: `font-family: Roboto, 'Arial', sans-serif;`.